en/05_Contributing/04_Release_Process.md

```diff
@@ -154,55 +154,75 @@ How to deprecate an API:
 
```

Comment:
Change the "How to deprecate an API" above into 
`### Deprecating API`

Reply:
Updated